common/core/gpudevice.cpp

```diff
@@ -202,6 +202,11 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
       UpdateDisplayState();
     }
   }
+  /* Adding sleep to avoid busy loop. The busy loop returns only
+   * if read() completes / fails. Race conditions can cause this
+   * busy loop hog the system. This 1 ms sleep can avoid that.
+   */
+  usleep(1000);
```

Comment:
@kalyankondapally  Sorry I missed this. I had tested with and without patch, there was no functional issue.
@tpalli I'm ok to drop this if there is a solution to avoid busy loop, please suggest and I can try that. We already discussed lseek one here https://github.com/01org/IA-Hardware-Composer/pull/60#discussion_r112874115